dspace-api/src/main/java/org/dspace/app/launcher/Readme.md

```diff
@@ -0,0 +1,57 @@
+# Adding new scripts to the dspace CLI 
+
+Scripts can be added to the script launcher using 2 mechanisms
+
+1. Defining a script in [dspace]/config/launcher.xml
+2. Creating a spring bean of type _org.dspace.app.launcher.CommandType_
+
+The advantage of the second mechanism is that a dependency can provide both the script and add it to the launcher without needing to modify the configuration of the receiving party.
+
```

Comment:
@rradillen , thanks for the clarification of the file name.  I changed the name of my test file to match your instructions above and the bean was found.  I recommend making that explicit in the documentation for those (like me) who are new to Spring.